conf/log4j.properties

```diff
@@ -0,0 +1,22 @@
+log4j.rootCategory=INFO, console
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.target=System.err
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
+log4j.logger.org.apache.zookeeper=ERROR
+log4j.logger.org.apache.hadoop.hdfs.distributedavatarfilesystem=ERROR
+log4j.logger.org.apache.hadoop.util.NativeCodeLoader=ERROR
+# Settings to quiet third party logs that are too verbose
+
+# Settings the HistoryServer logs
+#log4j.logger.net.qihoo.xlearning.jobhistory=DEBUG,RFA
```

Comment:
Please delete all these commented-out settings

Reply:
This is from older commits (when we were considering forking Qihoo's repo). I'm gonna remove them in the next several commits.